### ISSUE TITLE
Docker build runs from wrong base directory so it will fail on command `ADD`

### ISSUE THREAD

**Issue by JeroenKnoops** [philips-software/docker-ci-scripts#43]
# Problem
When adding a file (f.e. `entrypoint.sh`) to your image, this will fail.

# Example
https://github.com/philips-software/docker-blackduck/pull/60/checks?check_run_id=1110515797

# Version
This happens since version 2.3.0

**Comment by JeroenKnoops** [comment]
I think it's better to change the dockerfiles instead of trying to fix this here.
I think the previous way of working was wrong.

When you have a directory structure with multiple dockerfiles, you would expect to run the `docker build` command from the root of the project, not from each directory.

Imagine this:
```
.
├── 6
│   ├── dotnetcore-2.2.110
│   │   ├── Dockerfile
│   │   └── scripts
│   │       └── entrypoint.sh
│   ├── dotnetcore-3.0.101
│   │   ├── Dockerfile
│   │   └── scripts
│   │       └── entrypoint.sh
│   ├── dotnetcore-3.1.102
│   │   ├── Dockerfile
│   │   └── scripts
│   │       └── entrypoint.sh
│   ├── golang
│   │   └── Dockerfile
│   ├── java
│   │   ├── Dockerfile
│   │   ├── REPO
│   │   ├── TAGS
│   │   └── scripts
│   │       └── entrypoint.sh
│   └── node
│       └── Dockerfile
```

This now fails when the `entrypoint.sh` scripts have to be added.
This is because the previous version of `docker-ci-scripts` v2.2.2 expected to run `docker build` from the directory with the Dockerfile instead of from the root.

v2.2.2
```
$ cd 6/java
$ docker build . -t java
```

vs

v2.3.0
```
$ docker build -f 6/java/Dockerfile -t java
```

I want to break the API and change this so the last version is the correct behaviour. 
This means that people need to change their Dockerfiles when using `ADD` or `COPY`

Maybe we can add a `buildpath` optional argument when people don't want to change it.

**Comment by JeroenKnoops** [comment]
This is resolved in v3.0.0

I've added a BREAKING CHANGE section in the CHANGELOG.
